index.js

```diff
@@ -21,19 +30,27 @@ app.get('/game', (req, res) => {
  |         /     \\
  |
 `
-    })
+    };
+    
+    MapDataGames.set(initGame.id, initGame);
+    WriteObjectToJson({path:'./assets/saved-games.json'}, ConvertJson.strMapToObject(MapDataGames));
+    res.send(initGame);    
 })
 
 app.get('/game', (req, res) => {
-    Game.create()
+    
+    var dato =Game.create()
         .then(game => {
+            MapDataGames.set(game.id, game);    
+            WordReader.WriteObjectToJson({path:'./assets/saved-games.json'}, ConvertJson.strMapToObject(MapDataGames));

```

Comment:
the instruction said that the game should be saved in the file in the Game.create function and after saving the game we should return the object